requirements.txt

```diff
@@ -1,8 +1,13 @@
-    pytket == 1.4.1
+[comment]: # if version are updated, update the version in the build-manual file
+[comment]: # and the check-examples.yml, too.
```

Comment:
Don't need to update `check-examples.yml`.

Reply:
yes, thank you. Done